### ISSUE TITLE
First parameter indentation rule

### ISSUE THREAD

**Issue by ka8725** [bbatsov/rubocop#2625]
I like the following Ruby format to pass parameters into methods:

``` ruby
price = default_price * NightsCalculator.new(
                          start_date,
                          end_date).nights_count
```

but the `FirstParameterIndentation` style rule complaints about this. It assumes that only the following variants are possible:

``` ruby
price = default_price * NightsCalculator.new(start_date,
                                             end_date).nights_count
```

and 

``` ruby
price = default_price * NightsCalculator.new(
  start_date,
  end_date).nights_count
```

But the first one requires more symbols per line. And the second one is not readable in my mind.

What do you think is it make sense to pass code from the my preference?

**Comment by alexdowad** [comment]
We have another open issue for the same. It will be resolved soon.

**Comment by alexdowad** [comment]
Did we ever clear this one?

**Comment by ka8725** [comment]
I couldn't find the task you are referring to. If it's already done close this one.

**Comment by jonas054** [comment]
> Did we ever clear this one?

As far as I can see we didn't.

**Comment by alexdowad** [comment]
So what should this style be called?

**Comment by jonas054** [comment]
Fixed by #5699.
